net/netcheck/netcheck.go

```diff
@@ -389,7 +388,7 @@ const numIncrementalRegions = 3
 
 // makeProbePlan generates the probe plan for a DERPMap, given the most
 // recent report and whether IPv6 is configured on an interface.
-func makeProbePlan(dm *tailcfg.DERPMap, ifState *interfaces.State, last *Report) (plan probePlan) {
+func makeProbePlan(dm *tailcfg.DERPMap, ifState *netmon.State, last *Report) (plan probePlan) {
```

Comment:
Maybe rename to netmon.InterfaceState?

Reply:
Part of the motivation of the move is that interfaces.State isn't just interface state, though... it also has routing state: the default route. And maybe more in the future (folding in the net/routetable stuff?). So just "State" (network state... interfaces + routes + whatever else) seemed fitting. And shorter.